### PR TITLE
Milestone 1: Portfolio: setup and mobile first

### PR DESCRIPTION
**Pull request by gonzaloalonzo**
- Added linters files.
- Created Index according to the Model propose.
- Created Css file to make a responsive web page.

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @gonzaloalonzo 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that:

1. Not all the linters checks passed 
2. The pull request title is not good and doesn't reflect the changes made in the branch
3. The pull request summary is not descriptive. The pull request summary should describe changes that were introduced in this branch or features that were implemented. 👍 😎 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @gonzaloalonzo , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: 

You have submitted a project, that has the following issues
 
- The title of the pull request is not descriptive, it does not inform someone looking at the pull request what the pull request is about. A good title for this pull request would be something like `Milestone 1: Portfolio: setup and mobile first`. Please rectify this.
 
- The summary of the pull request is not descriptive. A good pull request summary should describe the changes that were introduced in this branch. I think it would be better if you listed all the features you worked on in the pull request summary.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by gonzaloalonzo** [comment]
Don't understand this (A good pull request summary should describe the changes that were introduced in this branch)

**Review by souad988 — COMMENTED** [review]
Hi @gonzaloalonzo , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a good PR title as html and css are technologies that will be used in all steps of this project the title should describe the feature or bloc of work done in this milestone the summary you used for this PR is very short but meaningful it can be used as a title and then in the summary describe in few points the work exp: Add header with logo and hamberger button , Add headlines .... 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by ABDULALI3468 — COMMENTED** [review]
Hi @gonzaloalonzo , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🔴

## Problems and their solutions:-

Please consider adding a professional title and description of your PR which describes what you have done in the Pull Request (PR). In this case, it's not describing it clearly.

>   pull_request_desc

> Title can be following:- **Milestone 1: Portfolio: setup and mobile-first**



Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        src="https://kit.fontawesome.com/26c38390b4.js"
+        crossorigin="anonymous"
+        ></script>
+  
+  </head>
+  <body>
+    
+      <nav class="navbar">
+        <p class="brand-title">My logo</p>
+        <i class="fas fa-bars"></i>
```
Comment (iambenkis) [inline index.html]: - [x] Your logo looks good 👍  , but for  semantic reason, kindly use an `a` tag 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      </nav>
+     
+      <section class="presentation">
+        <h2>I’m Alexandra. Glad to see you!</h2>
+      </section>
+
+      <article class="job">
```
Comment (iambenkis) [inline index.html]: - [x] As a header of your page , kindly use  the `header` tag , 
- [x] Noe that the header contains the site's title or logo (or both) and the main site navigation 👍 , kindly update that.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      <section class="contact">
+        <p>LET’S CONNECT</p>
+      </section>
+      <section class="socialmedia">
+        <i class="fa-brands fa-twitter"></i>
+        <i class="fa-brands fa-linkedin"></i>
+        <i class="fa-solid fa-m"></i>
+        <i class="fa-brands fa-square-github"></i>
+        <i class="fa-regular fa-hand-peace"></i>
+      </section>
```
Comment (iambenkis) [inline index.html]: - [x] Kindly use the `ul` and the `li` to this list of icons  for semantic reason 👍

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @gonzaloalonzo   
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [x] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — COMMENTED** [review]
Hi @gonzaloalonzo  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have all the linters passed. Kindly make sure you fix all linters errors before submitting your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by MeqdamAlqudah — COMMENTED** [review]
Hi @gonzaloalonzo 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image

## Highlights 
:+1: design 
:+1: semantic tags

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**